packages/button_driver/include/hardware_test_button/hardware_test_button.py

```diff
@@ -0,0 +1,78 @@
+import rospy
+
+from button_driver import ButtonDriver
+from dt_duckiebot_hardware_tests import HardwareTest, HardwareTestJsonParamType
+
+
+class HardwareTestButton(HardwareTest):
+    def __init__(
+        self,
+        driver: ButtonDriver,
+        led_blink_secs: int = 3,
+        led_blink_hz: int = 1,
+    ) -> None:
+        super().__init__()
+
+        # attr
+        self._driver = driver
+        self._button_released = False
+
+        # test settings
+        self.led_blink_secs = led_blink_secs
+        self.led_blink_hz = led_blink_hz
+
+    def test_id(self) -> str:
+        return "Top button"
+
+    def test_description_preparation(self) -> str:
+        return self.html_util_ul(
+            [
+                "Put your Duckiebot in its normal orientation, and make sure you can see and press the top button."
```

Comment:
Place your Duckiebot on a flat surface and locate the power button on the top plate.